manuscript/3-SoftConstr.txt

```diff
@@ -18,49 +18,48 @@
+Τα όρια μεταξύ σχεδίασης, κατασκευής και ελέγχου (αν υπάρχουν) ποικίλλουν, 
+καθώς εξαρτώνται από τις διεργασίες του κύκλου ζωής του λογισμικού που χρησιμοποιείται σε ένα έργο.
+
 Αν και είναι πιθανό να εφαρμοστεί λεπτομερή σχεδίαση πριν το στάδιο
-της κατασκευής του λογισμικού, κατά τη διάρκεια της κατασκευής γίνεται
-επίσης πολλή δουλειά αναφορικά με τη σχεδίαση.
+της κατασκευής του λογισμικού, κατά τη διάρκεια της κατασκευής δίνεται μεγάλη έμφαση στο κομμάτι της σχεδίασης.
 Γι' αυτό το λόγο, το γνωστικό πεδίο της κατασκευής λογισμικού είναι
 άρρηκτα συνδεδεμένο με αυτό της σχεδίασης λογισμικού.
 
 Κατά την κατασκευή,  οι μηχανικοί λογισμικού κάνουν τόσο δοκιμές μονάδων
 όσο και δοκιμές ολοκλήρωσης της δουλειάς τους.
-Γι' αυτό το λόγο και το γνωστικό πεδίο της κατασκευής λογισμικού είναι
-άρρηκτα συνδεδεμένο με αυτό του ελέγχου του.
+Γι' αυτό το λόγο το γνωστικό πεδίο της κατασκευής λογισμικού είναι
+άρρηκτα συνδεδεμένο και με αυτό του ελέγχου του.
+
+Κατά τη διάρκεια της κατασκευής, οι μηχανικοί λογισμικού πραγματοποιούν τόσο μοναδικές 
+όσο και ολοκληρωτικές δοκιμές του έργου τους. Άρα το γνωστικό πεδίο κατασκευής είναι 
+επίσης στενά συνδεδεμένη με το γνωστικό πεδίο ελέγχου.
 
 Η κατασκευή λογισμικού τυπικά παράγει το μεγαλύτερο αριθμό από αντικείμενα
 προσαρμογής, που πρέπει να διαχειριστούν μέσα σε ένα έργο λογισμικού
-(αρχεία με πηγαίο κώδικα, έγγραφο τεκμηρίωσης, σενάρια ελέγχου, κ.λπ.).
-Γι' αυτό το λόγο το γνωστικό πεδίο είναι άρρηκτα συνδεδεμένο με αυτό
-της διαχείρισης λογισμικού.
+(αρχεία με πηγαίο κώδικα, έγγραφο τεκμηρίωσης, σενάρια ελέγχου, κ.α.).
+Επομένως το γνωστικό πεδίο συνδέεται και με αυτό της διαχείρισης λογισμικού.
 
 Καθώς η ποιότητα λογισμικού είναι σημαντική σε όλα τα γνωστικά πεδία,
-ο κώδικας είναι το απόλυτο παραδοτέο ενός έργου λογισμικού και γι'
-αυτό το λόγο το γνωστικό πεδίο Ποιότητα λογισμικού είναι άρρηκτα
-συνδεδεμένο με αυτό της κατασκευής λογισμικού.
+ο κώδικας είναι το απόλυτο παραδοτέο ενός έργου λογισμικού και γι'αυτό το λόγο 
+το γνωστικό πεδίο ποιότητας λογισμικού είναι άρρηκτα συνδεδεμένο με αυτό της κατασκευής λογισμικού.
 
 Εφόσον, η κατασκευή λογισμικού απαιτεί γνώση αλγορίθμων και προγραμματιστικές
 πρακτικές, το γνωστικό πεδίο της κατασκευής συνδέεται και με το πεδίο
-γνώσης της θεμελιώδους υπολογιστικής θεωρίας, το οποίο ασχολείται
-με την επιστήμη των υπολογιστών, που υποστηρίζει το σχεδιασμό και
-την κατασκευή προϊόντων λογισμικού.
-Είναι ακόμα συνδεδεμένο και με τη διοίκηση έργων, στο βαθμό που η
-διοίκηση της κατασκευής μπορεί να παρουσιάσει αξιόλογες προκλήσεις.
+γνώσης της θεμελιώδους υπολογιστικής θεωρίας, με το οποίο ασχολείται
+η επιστήμη των υπολογιστών, που υποστηρίζει το σχεδιασμό και
+την κατασκευή προϊόντων λογισμικού. Τέλος είναι συνδεδεμένο και με τη διοίκηση έργων, 
+στο βαθμό που η διοίκηση της κατασκευής μπορεί να παρουσιάσει αξιόλογες προκλήσεις.
 
 ## Κατανομή των θεμάτων για την Κατασκευή λογισμικού
```

Comment:
The breakdown of topics for the Software
Architecture KA is shown in Figure 4-1. 
Πρέπει επίσης να μεταφραστεί